File: quivi.pos.react.app/src/hooks/api/Dtos/printers/Printer.ts
export interface Printer {
    readonly id: string;
    readonly name: string;
    readonly printConsumerInvoice: boolean;
    readonly printConsumerBill: boolean;
    readonly sendOrdersToToKitchen: boolean;
    readonly canOpenCashDrawer: boolean;
    readonly canPrintCloseDayTotals: boolean;
}

File: Quivi.Pos.Api/Dtos/Printer.cs
namespace Quivi.Pos.Api.Dtos
{
    public class Printer
    {
        public required string Id { get; set; }
        public required string Name { get; set; }
        public bool PrintConsumerInvoice { get; set; }
        public bool PrintPreparationRequest { get; set; }
        public bool PrintConsumerBill { get; set; }
        public bool CanOpenCashDrawer { get; set; }
        public bool CanPrintCloseDayTotals { get; set; }
    }
}